store/tikv/2pc.go

```diff
@@ -1644,7 +1644,7 @@ func (batchExe *batchExecutor) process(batches []batchMutations) error {
 		}
 	}
 	close(exitCh)
-	metrics.TiKVTokenWaitDuration.Observe(batchExe.tokenWaitDuration.Seconds())
+	metrics.TiKVTokenWaitDuration.Observe(float64(batchExe.tokenWaitDuration.Nanoseconds()))
```

Comment:
The `tokenWaitDuration` is a duration type which has nano second count, but in https://github.com/pingcap/tidb/pull/17268 the observed value is changed to seconds(see the grafana picture in 17268). I'm a bit confused which is correct.

Reply:
The bucket is defined to be `Buckets:   prometheus.ExponentialBuckets(1, 2, 34), // 1ns ~ 8s`. The base is 1, so it should be a nanosecond.

Reply:
It's expected to be observed in nanoseconds. #17268 doesn't fix this. The root cause is the mismatch between the metric and the grafana panel.
```go
	TiKVTokenWaitDuration = prometheus.NewHistogram(
		prometheus.HistogramOpts{
			Namespace: namespace,
			Subsystem: subsystem,
			Name:      "batch_executor_token_wait_duration",
			Buckets:   prometheus.ExponentialBuckets(1, 2, 34), // 1ns ~ 8s
			Help:      "tidb txn token wait duration to process batches",
		})
```